src/routes/Profile/Competences/constants.ts

```diff
@@ -14,6 +14,7 @@ export interface FilterData {
 
 export type FiltersType = Record<OsaaminenLahdeTyyppi, FilterData[]>;
 export const FILTERS_ORDER = ['TOIMENKUVA', 'KOULUTUS', 'PATEVYYS', 'KIINNOSTUS', 'MUU_OSAAMINEN'] as const;
```

Comment:
If I understand the use of `FILTERS_ORDERS` it should not have the 'KIINNOSTUS' as one of the values?

Reply:
I think ´KIINNOSTUS´ is used in backend and is used somewhere in the system, but in this case the competences -page uses only these 4 competences.  🤷

Reply:
Let's keep this for now and if needed, refactoring this later on a separate ticket.